bip-0085.mediawiki

```diff
@@ -248,7 +261,7 @@ INPUT:
 * PATH: m/83696968'/128169'/64'/0'
 
 OUTPUT
-* DERIVED ENTROPY=492db4698cf3b73a5a24998aa3e9d7fa96275d85724a91e71aa2d645442f878555d078fd1f1f67e368976f04137b1f7a0d19232136ca50c44614af72b5582a5c
+* DERIVED ENTROPY=74a2e87a9ba0cdd549bdd2f9ea880d554c6c355b08ed25088cfa88f3f1c4f74632b652fd4a8f5fda43074c6f6964a3753b08bb5210c8f5e75c07a4c2a20bf6e9
```

Comment:
this is wrong, how did you calculate it ? 

In your previous PR https://github.com/bitcoin/bips/pull/1600/files you have correctly fixed bug (incorrect entropy) in Base64 pwd application

here you're fixing hex app that is not broken and correct value MUST be 492db4698cf3b73a5a24998aa3e9d7fa96275d85724a91e71aa2d645442f878555d078fd1f1f67e368976f04137b1f7a0d19232136ca50c44614af72b5582a5c (no change here)

Reply:
Let me show you the output of the reference implementation.
```
bipsea derive -x xprv9s21ZrQH143K2LBWUUQRFXhucrQqBpKdRRxNVq2zBqsx8HVqFk2uYo8kmbaLLHRdqtQpUm98uKfu3vca1LqdGhUtyoFnCNkfmXRyPXLjbKb -a base64 -n 21 -d
dKLoepugzdVJvdL56ogNV
entropy: 74a2e87a9ba0cdd549bdd2f9ea880d554c6c355b08ed25088cfa88f3f1c4f74632b652fd4a8f5fda43074c6f6964a3753b08bb5210c8f5e75c07a4c2a20bf6e9
```
I can publish a `-d` debug mode to PyPI if you want to see it.

Also note that [this test](https://github.com/akarve/bipsea/pull/66/files#diff-8908e363d26fecf028c8662410875b07c671a7b7f5b0f236378523004a67d0b6R67) is an xfail.

I also note that in #1600 the change was indeed to the [hex derived entropy](https://github.com/bitcoin/bips/pull/1600/files#diff-eae7a61b6d2a0b6091c1ce04618f216cbb871d987b985ba712de89e04014654cR314).

Reply:
Thank you for catching that. I didn't read the diff properly. This is for Base64:
```
bipsea derive -x xprv9s21ZrQH143K2LBWUUQRFXhucrQqBpKdRRxNVq2zBqsx8HVqFk2uYo8kmbaLLHRdqtQpUm98uKfu3vca1LqdGhUtyoFnCNkfmXRyPXLjbKb -a base64 -n 21 -d
dKLoepugzdVJvdL56ogNV
entropy: 74a2e87a9ba0cdd549bdd2f9ea880d554c6c355b08ed25088cfa88f3f1c4f74632b652fd4a8f5fda43074c6f6964a3753b08bb5210c8f5e75c07a4c2a20bf6e9
```